l10n_it_ateco/model/partner.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2014 Abstract (<http://abstract.it>).
+# Copyright (C) 2016 Ciro Urselli (<http://www.apuliasoftware.it>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+
+    _inherit = "res.partner"
+
+    ateco_category_ids = fields.Many2many(
```

Comment:
Ciao, in base alla discussione in call, i campi andrebbero rinominati col prefisso l10n_it_  come fa Odoo.